Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -476,6 +476,26 @@ float Renderer::CalculateDrawAspectRatio(int target_width, int target_height)
   }
 }
 
+void Renderer::ScaleToDisplayAspectRatio(const int width, const int height, float* scaledWidth,
```

Comment:
You can make this return a `std::pair` or a `std::tuple` instead of using out parameters. Unpacking can then be utilized like so.

```cpp
std::tie(scaled_width, scaled_height) = ScaleToDisplayAspectRatio(width, height);
```

This would also make the function forward compatible with C++17's [structured bindings](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0144r2.pdf)